client/packages/system/src/Patient/Insurance/components/InsuranceStatusToggle/InsuranceStatusToggle.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, ReactElement } from 'react';
+import { Checkbox, Typography } from '@common/components';
+import { useTranslation } from '@common/intl';
+
+import { Box } from '@openmsupply-client/common';
+
+interface InsuranceStatusToggleProps {
+  isActive: boolean;
+  onChange: (isActive: boolean) => void;
+}
+
+export const InsuranceStatusToggle: FC<InsuranceStatusToggleProps> = ({
+  isActive,
+  onChange,
+}): ReactElement => {
+  const t = useTranslation();
+  return (
+    <Box pt={2}>
+      <Typography variant="body1">{t('label.is-active-insurance')}</Typography>
+      <Box sx={{ gap: 2, display: 'flex', flexDirection: 'row' }}>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'center',
+          }}
+        >
+          <Checkbox checked={isActive} onChange={() => onChange(true)} />
+          <Typography variant="body1">{t('label.active')}</Typography>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'center',
+          }}
+        >
+          <Checkbox checked={!isActive} onChange={() => onChange(false)} />
+          <Typography variant="body1">{t('label.inactive')}</Typography>
```

Comment:
Just wondering why two checkboxes, as opposed to one (where "checked" is active and vice versa)? I think the general UI trend is to use a radio button for multiple exclusive options like this, but I think a single checkbox would do the job here. Was there a UI guide you were working off?